packages/app/package.json

```
@@ -16,6 +16,8 @@
     "@feathersjs/socketio-client": "4.5.11",
     "@fortawesome/fontawesome-free": "^5.15.1",
     "@project-leek/commons": "*",
+    "typeface-baloo": "^1.1.13",
+    "typeface-roboto": "^1.1.13",
```

Please pin dependencies and run `yarn` again to update `yarn.lock`

```suggestion
    "typeface-baloo": "1.1.13",
    "typeface-roboto": "1.1.13",
```

---

fixed in f513c5e0436795129e3f0e0f75e7d59c6dc1f421